src/main/sensors/gyro.h

```diff
@@ -73,6 +73,8 @@ typedef struct gyroConfig_s {
     uint16_t gyro_kalman_q;
     uint16_t gyro_kalman_r;
     uint16_t gyro_kalman_p;
+    uint16_t gyro_bqrcf2_q;
```

Comment:
@fujin: What do you think now about changing these to reuse `gyro_kalman_q`, `gyro_kalman_r`, to make it possible for users to switch between the two filter types without resetting their config?